lib/hacluster.pm

```diff
@@ -979,6 +980,44 @@ sub activate_ntp {
     systemctl "enable --now $ntp_service.service";
 }
 
+=head2 script_output_retry_check
+
+  script_output_retry_check(cmd=>$cmd, regex_string=>$regex_sring, [retry=>$retry, sleep=>$sleep, ignore_failure=>$ignore_failure]);
+
+Executes command via 'script_output' subroutine and makes a sanity check against a regular expression. Command output is returned
+after success, otherwise the command is retried defined number of times. Test dies after last unsuccessfull retry.
+
+C<$cmd> command being executed.
+C<$regex_string> regular expression to check output against.
+C<$retry> number of retries. Defaults to C<5>.
+C<$sleep> sleep time between retries. Defaults to C<10s>.
+C<$ignore_failure> do not kill the test upon failure.
+
+  Example: script_output_retry_check(cmd=>'hostname', regex_string=>'^node01$', retry=>'100', sleep=>'60', ignore_failure=>'1');
+
+=cut
+
+sub script_output_retry_check {
```

Comment:
Wonder if it would make more sense to move this to `lib/utils.pm` next to the other similar functions.